core-tests/src/test/java/org/mule/runtime/core/el/DefaultExpressionModuleBuilderTestCase.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.el;
+
+import org.junit.Test;
+import org.mule.runtime.api.el.BindingContext;
+import org.mule.runtime.api.el.ExpressionModule;
+import org.mule.runtime.api.el.ModuleNamespace;
+import org.mule.runtime.api.metadata.TypedValue;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+
+import static org.hamcrest.Matchers.*;
```

Comment:
We have a pending task of improving the formatter checks, but for now some of the checks still have to be enforced in the PRs. I'll make sure to update our guidelines so it's at least reflected there.